decidim-core/spec/queries/decidim/public_activities_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Decidim::PublicActivities do
```

Comment:
We have to write spec for `Decidim::ActivitySearch` instead.

The name has been changed after the Searchlight removal. There may be also some other changes we need to do here.